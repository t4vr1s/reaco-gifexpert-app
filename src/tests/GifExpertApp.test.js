import '@testing-library/jest-dom'
import React from 'react'
import { shallow } from 'enzyme'
import GifExpertApp from '../GifExpertApp'

/* eslint-disable */
describe('pruebas al componente <GifExpertApp/>', () => {
  test('snapshot al componente', () => {
    const wrapper = shallow(<GifExpertApp />)
    expect(wrapper).toMatchSnapshot()
  })

  test('should mostrar una lista de componentes', () => {
    const categories = ['Killua', 'Mob', 'Saitama']
    const wrapper = shallow(<GifExpertApp defaultCategories={categories} />)
    expect(wrapper).toMatchSnapshot()
    expect(wrapper.find('GifGrid').length).toBe(categories.length)
  })
})
