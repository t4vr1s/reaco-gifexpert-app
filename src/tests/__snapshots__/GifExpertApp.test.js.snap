// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas al componente <GifExpertApp/> should mostrar una lista de componentes 1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol>
    <GifGrid
      category="Killua"
      key="Killua"
    />
    <GifGrid
      category="Mob"
      key="Mob"
    />
    <GifGrid
      category="Saitama"
      key="Saitama"
    />
  </ol>
</Fragment>
`;

exports[`pruebas al componente <GifExpertApp/> snapshot al componente 1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol />
</Fragment>
`;
